src/components/ItemDetail/use/useLogs.ts

```diff
@@ -0,0 +1,51 @@
+import { computed, ComputedRef } from 'vue'
+import { Log } from '/@/lib/apis'
+import { stringifyTimeByDate } from '/@/lib/date'
+
+export interface LogSummary {
+  id: number
+  userName: string
+  text: string
+}
+
+const traP_ID = 1
+
+const useLogs = (props: {
+  logs: Log[]
+}): {
+  logSummaries: ComputedRef<LogSummary[]>
+} => {
+  const logSummaries = computed(() =>
+    props.logs.map(v => {
+      const userName = v.user.name
+      const ownerWord =
+        v.ownerId === traP_ID || v.type > 1 ? '' : `${v.owner.name}さんの`

```

Comment:
```suggestion
        v.ownerId === traP_ID || v.type === LogType.add || v.type === LogType.remove ? '' : `${v.owner.name}さんの`
```
`enum`は数値で扱わないほうがよさ
(`LogType`の`import`が必要)

Reply:
今のところ`LogType`のなかに`add`と`remove`がないからこうしたたんだけど先にbooQリポジトリのswagger変更入れちゃったらいい？

Reply:
そうだね
向こうマージしちゃってよいと思う 🙏